src/EnergyPlus/Data/EnergyPlusData.hh

```diff
@@ -375,6 +376,7 @@ struct EnergyPlusData : BaseGlobalStruct
     std::unique_ptr<DesiccantDehumidifiersData> dataDesiccantDehumidifiers;
     std::unique_ptr<DisplacementVentMgrData> dataDispVentMgr;
     std::unique_ptr<DualDuctData> dataDualDuct;
+    std::unique_ptr<EIRFuelFiredHeatPumpsData> dataEIRFuelFiredHeatPump;
```

Comment:
Can someone explain why these things need to be `unique_ptr` as opposed to just a normal pointer? Because `unique_ptr`'s are hipster?

Reply:
Assuming this was explained and would be some future effort all together with other similar unique pointers?

Reply:
Yes, this change would be galaxies away from the scope of this PR.  Don't worry about this here.  Thanks!